package.json

```diff
@@ -17,7 +17,7 @@
   "scripts": {
     "pretest": "./scripts/build_peg.sh",
     "test": "./scripts/run_tests.sh",
-    "lint:js": "eslint --max-warnings=0 ./src/ ./examples/ ./test/unit_tests ./index.js -f table --ext .js --ext .jsx",
+    "lint:js": "eslint --max-warnings=0 ./src/ ./examples/ ./test/unit_tests ./index.js -f table --ext .js --ext .jsx --fix",
```

Comment:
why? this will mess up the CI cause people can commit unformatted code and the linter will fix it and pass.
you can add a `lint:js:fix`